src/libYARP_manager/include/yarp/manager/xmlclusterloader.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2017 iCub Facility
+ * Authors: Nicolo' Genesio <nicolo.genesio@iit.it>
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ */
+
+
+#ifndef YARP_MANAGER_XMLCLUSTERLOADER
+#define YARP_MANAGER_XMLCLUSTERLOADER
+
+#include <yarp/manager/ymm-types.h>
+#include <yarp/manager/manifestloader.h>
+
+
+
+namespace yarp {
+namespace manager {
+
+struct ClusNode
```

Comment:
I think that saving the `ter` from cluster here is not really worth, I'd rename it `ClusterNode`